from utils.data_management import reset

reset()
